csv.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+# Reading from CSV files
+class Csv
+  def load_data(filename)
+    CSV.read(filename)
```

Comment:
enclose in begin end block

Reply:
It shows an underline when I enclose it in begin block. Says "redundant begin block". And rubocop auto corrected it with the command rubocop --auto-correct-all